### ISSUE TITLE
Overwriting the pageUid depending on the application contex

### ISSUE THREAD

**Issue by rboenischAqua** [tritum/form_element_linked_checkbox#59]
Hello all,

I would like to know if it is possible to overwrite the pageUid depending on the application context?
I tried with variants, which does not work.
Here is an excerpt from my yaml file.
```
      -
        properties:
          pageUid: '1418'
          linkText: 'Teilnahmebedingungen! '
          elementClassAttribute: small
          fluidAdditionalAttributes:
            required: required
        type: LinkedCheckbox
        identifier: linkedcheckbox-1
        label: 'Ich akzeptiere die'
        variants:
          -
            identifier: varient-production
            condition: 'applicationContext matches "#Production#"'
            properties:
              pageUid: '1423'
          -
            identifier: varient-test
            condition: 'applicationContext matches "#Testing#"'
            properties:
              pageUid: '1418'
        validators:
          -
            identifier: NotEmpty
```

I'm using Typo3 11.5.27.

Thanks in advance!

**Comment by eliashaeussler** [comment]
Hi @rboenischAqua, this will be possible once #53 is merged and released.

**Comment by tritum** [comment]
The PR has been merged. By pulling latest master you can test @eliashaeussler's solution. Thanks.

**Comment by tritum** [comment]
I've released v4.0 including this patch.
